api/views/question.py

```diff
@@ -79,7 +79,7 @@ def get_favorite_count(self, request, pk=None):
         return Response(uqj_count)
 
     # TODO: Fix permissions for this
-    @action(detail=True, methods=['post'], url_path='opened-question')
+    @action(detail=True, methods=['get'], url_path='opened-question')
```

Comment:
dont' change the method.

Reply:
I tried with post but it actually doesn't work, it says the method is not allowed